lib/display.rb

```diff
@@ -0,0 +1,12 @@
+class Display
+  def display_grid(grid)
```

Comment:
Question: What are you displaying? A TTT board or a grid?

Reply:
Good question! I'll change this to say board to make it clearer